recipes/mold/1.4.x/test_package/conanfile.py

```diff
@@ -1,20 +1,17 @@
 import os
 from conan import ConanFile
-from conan.tools.build import cross_building
-from conan.tools.layout import basic_layout
+from conan.tools.build import can_run
 
 
 class MoldTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
-    # VirtualRunEnv can be avoided if "tools.env.virtualenv:auto_use" is defined
-    # (it will be defined in Conan 2.0)
-    generators = "VirtualBuildEnv"
```

Comment:
@uilianries @prince-chrismc What sould it be in the end? VirtualBuildEnv or VirtualRunEnv? I am confused

Reply:
It stopped working, as I have applied a suggestion with this change.
Why on your oppinion, it should be Run environment and not build?